rewrite-gradle/src/main/java/org/openrewrite/gradle/UpgradeDependencyVersion.java

```diff
@@ -208,18 +211,15 @@ public J visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx)
                         if(acc.gaToNewVersion.containsKey(ga)) {
                             return m;
                         }
-                        String oldVersion = findCurrentResolvedVersion(groupId, artifactId, m.getSimpleName(), gradleProject);
```

Comment:
This in particular was completely preventing updating the dependencies contained within the subproject because they don't exist in the current project's `GradleProject` marker.